### DISCUSSION TITLE
Nav to external URL in the same tab

### DISCUSSION THREAD

**Discussion by lancemao** [vuejs/vitepress#1211] [General]
In my nav, there are some external URL, let's take the official tutorial for example:

nav: [
      { text: 'Guide', link: '/guide/' },
      { text: 'Changelog', link: 'https://github.com' },
]

Now, after click 'Changelog', browser opens a new tab for https://github.com
However, I want https://github.com to be opened in the current tab.
Is it possible to do so?

**Comment by MarkusKeck** [comment]
Maybe this helps https://github.com/vuejs/vitepress/discussions/1015
